src/super_gradients/training/models/detection_models/yolo_nas/dfl_heads.py

```diff
@@ -263,10 +263,10 @@ def out_channels(self):
         return None
 
     def forward(self, feats: Tuple[Tensor]):
-        if self.training:
-            return self.forward_train(feats)
-        else:
-            return self.forward_eval(feats)
+        # if self.training:
```

Comment:
Decomment please.

Reply:
Fixed in 3073d4b95173e24388ed7481fd4a4bd72816b4a8, thanks for spotting.
Not sure how it leaked to this branch, prob. by misclick